### ISSUE TITLE
Optifabric isn't working with origins mod 1.17.1

### ISSUE THREAD

**Issue by MysterymanPJ** [Chocohead/OptiFabric#681]
I have optifine and optifabric in a mods folder with fabric api and the origins mod and the game crashes when i try to load it

**Comment by shiniebub** [comment]
sorry that im commenting without an answer to the problem, but i just want to make it known i also have this issue.

**Comment by snrSavage666** [comment]
yeah apparently origins is not compatible with any version of optifabric does somebody know if there's any way to fix this without having to remove optifabric
